cmd/cryptdecode/cryptdecode.go

```diff
@@ -39,40 +38,29 @@ use it like this
 `,
 	Run: func(command *cobra.Command, args []string) {
 		cmd.CheckArgs(2, 11, command, args)
-		fsrc := cmd.NewFsSrc(args)
-		if Reverse {
-			cmd.Run(false, false, command, func() error {
-				return cryptEncode(fsrc, args[1:])
-			})
-		} else {
-			cmd.Run(false, false, command, func() error {
-				return cryptDecode(fsrc, args[1:])
-			})
-		}
+		cmd.Run(false, false, command, func() error {
+			_, configName, _, err := fs.ParseRemote(args[0])
+			if err != nil {
+				return err
+			}
+			cipher, err := crypt.NewCipher(configName)
```

Comment:
What does this do if you give it a remote which isn't a crypt remote?

Maybe check `config.FileGet(configName, "remote") == "crypt"` and return a nice error if not?

Reply:
This is a good idea. I changed the returned error from
```plain
2018/02/24 23:36:47 Failed to cryptdecode: password not set in config file
```
to
```plain
2018/02/24 23:49:05 Failed to cryptdecode: The remote needs to be of type "crypt"
```